### PR TITLE
Introduce adrs

### PR DESCRIPTION
**Pull request by tw-jeff-burroughs**

### CODE REVIEW

**Comment by tw-jeff-burroughs** [comment]
This handles issue #57 https://github.com/invinst/invisible-flow/issues/57

File: architecture/adr-001-use-python.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Use Python
+
+## Status
+
+accepted
+
+## Context
+
+Part of the Invisible Flow project is to take data in one format and produce output in 
+varying formats.  With Python's large libraries set for data manipulation and community
```
Comment (adesca) [inline architecture/adr-001-use-python.md]: "With Python's large set of libraries" or "library set"

Comment (adesca) [reply architecture/adr-001-use-python.md]: *"large library set"

File: architecture/adr-002-use-gcs.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Use Google Cloud Services for storage
+
+## Status
+
+accepted
+
+## Context
+
+THe need here is to be able to store the transformed data in the cloud for access later.
```
Comment (adesca) [inline architecture/adr-002-use-gcs.md]: "The" instead of "THe"

**Pull request merged**